Dockerfile

```diff
@@ -9,21 +9,13 @@ RUN echo "@edge http://dl-4.alpinelinux.org/alpine/edge/main" | tee -a /etc/apk/
  && apk -U --no-progress add ca-certificates bash git linux-pam s6@edge curl openssh socat \
  && chmod +x /usr/sbin/gosu
 
-# Configure Go and build Gogs
-ENV GOPATH /tmp/go
-ENV PATH $PATH:$GOPATH/bin
+ENV GOGS_CUSTOM /data/gogs
 
 COPY . /app/gogs/
 WORKDIR /app/gogs/
 RUN ./docker/build.sh
 
-ENV GOGS_CUSTOM /data/gogs
-
-# Create git user for Gogs
-RUN adduser -D -g 'Gogs Git User' git -h /data/git/ -s /bin/sh && passwd -u git
-RUN echo "export GOGS_CUSTOM=/data/gogs" >> /etc/profile
-
 # Configure Docker Container
 VOLUME ["/data"]
 EXPOSE 22 3000
-CMD ["./docker/start.sh"]
+CMD ["docker/start.sh"]
```

Comment:
Only thing unsure is this, can you confirm `"docker/start.sh"` does exactly the same thing as `./docker/start.sh` does?